gorgias/gtm-chat
# [SS-367]: Publish a GTM Template for the chat

### Linear issue : <a href="https://linear.app/gorgias/issue/SS-367/publish-a-gtm-template-for-gorgias-chat">SS-367</a>

Docs to publish a GTM Template : https://developers.google.com/tag-manager/templates/gallery

### Changes
- Add a GTM template file
- Add a metadata.yaml file (to update once template is ready)
- Add an Apache 2 license

### Tests (With staging)
- The tag used for this test has been created from the GTM Template in https://tagmanager.google.com/?authuser=1#/container/accounts/6003048376/containers/39256078/workspaces/19
- Make sure that your ad block is turned off
- Browse to https://gfay-gtm-test.myshopify.com/ (password: awnewd)
- The chat should be loaded via GTM

### TODO
- Update metadata.yaml with the release commit once ready
- Create a documentation on https://docs.gorgias.com/ to use the template
